### PR TITLE
Added tamil language translation

### PR DESCRIPTION
**Pull request by strikerrocker**

### CODE REVIEW

File: src/main/resources/assets/creepy-creepers/lang/ta-IN.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (UpcraftLP) [inline src/main/resources/assets/creepy-creepers/lang/ta-IN.json:1]: file name must be `lowercase_with_underscores`

**Review by UpcraftLP — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/creepy-creepers/lang/ta-IN.json (hunk 1 above)
Comment (strikerrocker) [reply src/main/resources/assets/creepy-creepers/lang/ta-IN.json]: done

File: src/main/resources/assets/creepy-creepers/lang/ta-in.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (UpcraftLP) [inline src/main/resources/assets/creepy-creepers/lang/ta-in.json:1]: file name must be underscores, not hyphens..

**Comment by ProxyNeko** [comment]
Looking good! Thanks for the PR!

File: src/main/resources/assets/creepy-creepers/lang/ta-in.json (hunk 2 above)
Comment (strikerrocker) [reply src/main/resources/assets/creepy-creepers/lang/ta-in.json:1]: #8

**Pull request merged**